Core.Collectors/Collector/CollectionNode.cs

```diff
@@ -49,7 +49,11 @@ protected CollectionNode()
         /// </summary>
         public Func<JObject, bool> HaltCollection { get; set; } = record => false;
 
-        public List<HttpResponseSignature> WhitelistedResponses = new List<HttpResponseSignature>();
+        public List<HttpResponseSignature> AllowlistedResponses = new List<HttpResponseSignature>();
+
+        [ObsoleteAttribute("This property is deprecated and will be removed in the next revision for compliance reasons. Use AllowlistedResponses instead.")]
```

Comment:
Nice! Honestly, I was not even thinking of adding this properly, but this is much better and would make history look better.